app/views/projects/show.html.erb

```diff
@@ -12,8 +12,25 @@
       .html_safe
   %>;
 
+  var taxon_lookup = <%=
```

Comment:
does it make sense to move this javascript out of this view and into one of these govuk_frontend_toolkit modules in assets/javascript?